src/containers/AddTodo.spec.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import { shallow } from 'enzyme';
+
+import AddTodo from './AddTodo';
+
+const setup = (setupProps = {}) => {
+  const store = configureStore()({ todos: [] });
+  const wrapper = shallow(<AddTodo store={store} />);
+
+  return {
+    store,
+    wrapper
+  };
+};
+
+describe('AddTodo', () => {
```

Comment:
Ok for the container like this, as my experience in current project, we usually organize the test like this:
```
describe('<AddTodo />', () => {
  test('should match snapshot', () => {
  });
  
  describe('handleChange', () => {
    test('should update value in state correctly', () => {
    });
  });
  
  describe('handleSubmit', () => {
    describe('when value is present', () => {
      it('should dispatch addTodo action with correct value and reset value in state', () => {
      });
    });
    
    describe('when value is empty', () => {
      test('should not do anything', () => {
      });
    });
  });
});
```